modules/cnv-importing-vmware-vm.adoc

```diff
@@ -70,7 +70,7 @@ If you do not select a storage class, {CNVProductName} uses the default storage
 
 . In the *Advanced* screen, enter the *Hostname* and *Authorized SSH Keys* if you are using `cloud-init`.
 +
-You must xref:../../../cnv/cnv_virtual_machines/cnv-edit-vms.adoc#cnv-editing-vm-yaml-web_cnv-edit-vms[edit the virtual machine's YAML configuration] after importing it:
+You must edit the virtual machine's YAML configuration after importing it:
```

Comment:
This part of the procedure (editing the VM yaml) should be removed. It was a workaround. The bug will be fixed when 4.4 is released.